### PR TITLE
[1.4-candidate]: dependencies upgrade, naming refactoring

### PR DESCRIPTION
**Pull request by andyislegend**

### CODE REVIEW

File: src/main/java/coresearch/cvurl/io/model/CvurlConfig.java [hunk 1]
```diff
@@ -27,7 +27,7 @@
 /**
  * Data class that holds configuration of {@link coresearch.cvurl.io.request.CVurl}
  */
-public class Configuration {
+public class CvurlConfig {
```
Comment (imahlatii) [inline src/main/java/coresearch/cvurl/io/model/CvurlConfig.java:30]: CVurlConfig?

Comment (andyislegend) [reply src/main/java/coresearch/cvurl/io/model/CvurlConfig.java:30]: yes

Comment (imahlatii) [reply src/main/java/coresearch/cvurl/io/model/CvurlConfig.java:30]: I mean, it's better to name this class _CVurlConfig_ instead of _CvurlConfig_

Comment (andyislegend) [reply src/main/java/coresearch/cvurl/io/model/CvurlConfig.java:30]: 2much upper cases, no?

**Pull request merged**